packages/salesforcedx-lightning-lsp-common/src/utils.ts

```diff
@@ -5,14 +5,16 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
-import * as jsonc from 'jsonc-parser';
 import { basename, extname, join, relative, resolve } from 'node:path';
 import { FileEvent, FileChangeType } from 'vscode-languageserver';
 import { TextDocument } from 'vscode-languageserver-textdocument';
 import { URI } from 'vscode-uri';
 import { BaseWorkspaceContext } from './baseContext';
 import { IFileSystemProvider } from './providers/fileSystemDataProvider';
 
+// Cache the dynamically imported tiny-jsonc module to avoid repeated imports
```

Comment:
node/browser runtime does the automatically for you, no need for us to do it manually